op-challenger/game/fault/trace/cannon/state_converter_test.go

```diff
@@ -48,4 +49,30 @@ func TestLoadState(t *testing.T) {
 		require.NoError(t, json.Unmarshal(testState, &expected))
 		require.Equal(t, &expected, state)
 	})
+
+	t.Run("Binary", func(t *testing.T) {
+		var expected singlethreaded.State
+		require.NoError(t, json.Unmarshal(testState, &expected))
```

Comment:
Just a note that [this testState](https://github.com/ethereum-optimism/optimism/blob/197e4f52dfdb86cf1dda5c2d5788c5fcbb4f1b48/op-challenger/game/fault/trace/cannon/test_data/state.json) has a lot of zeroed fields.  Should we add some more non-zero data?

Reply:
I think generally its ok to just be a basic smoke test given that we're reusing cannon's code to do the actual parsing - we really just want to make sure it can detect the file type correctly.  I've fleshed it out a little bit more though as that was quite minimalist, but its still very much a bare bones state.